environment.yml

```diff
@@ -19,11 +20,16 @@ dependencies:
   - pydantic
   - pymbar
   - python >= 3.6
+  - pymol
+  - rdkit
```

Comment:
Do we explicitly use rdkit?

Reply:
It looks like rdkit is used for `xcimporter.py` in the fragalysis api. I'll check if it is needed for this pipeline to work and remove it (and pymol) if not.

Reply:
Removed